src/components/Planner/Planner.tsx

```diff
@@ -1,37 +1,88 @@
 import React, { useEffect, useState } from "react";
-import { Box, Center, Grid, GridItem, Text } from "@chakra-ui/react";
+import {
+  Center,
+  Grid,
+  GridItem,
+  Text,
+  Image,
+  Container
+} from "@chakra-ui/react";
 import { DragDropContext, DropResult } from "react-beautiful-dnd";
 import { Droppable } from "react-beautiful-dnd";
 import { Draggable } from "react-beautiful-dnd";
 import { getRecipe } from "../../utils/fetchData";
-import { SavedRecipe } from "../../utils/types";
+import { PlannerRecipe } from "../../utils/types";
 
 const Planner = () => {
-  const [recipes, setRecipes] = useState<Array<SavedRecipe>>([]);
+  const [columns, setColumns] = useState({
```

Comment:
NON-BLOCKING: I might use more descriptive keys here (recipes, day1, day2, etc.). Also, you can define a type for this.

So, you can do:
```
type TColumns = {
recipes: Array<insert_type_here>,
etc.
};
const [columns, setColumns] = useState<TColumns>(...);
```